src/components/KTreeList/KTreeItem.vue

```diff
@@ -70,7 +71,16 @@ const slots = useSlots()
 
 const hasIcon = computed((): boolean => !props.hideIcons || !!slots['item-icon'])
 
-const handleClick = () => {
+const handleClick = (event: any) => {
+  if (event.target) {
+    const ignoredElements = 'a:not(.disabled), button:not([data-tree-item-trigger]):not(:disabled), label, input:not(:disabled), select:not(:disabled)'
```

Comment:
Sorry, there was a slight error in my previous comment. I forgot that disabled elements do not respond to click events and do not bubble up to the container and we are already inside a click handler here. Therefore, there's no need to add `:not(:disabled)` here (although `a:not(.disabled)` might still be necessary).

Reply:
I tested it in the sandbox and it works as expected without `:not(:disabled)`